src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -36,29 +36,12 @@
 */
 public class ADDR_Addresses_TDTM extends TDTM_Runnable {
```

Comment:
* Classes must use `with sharing` and never omit the sharing clause
* Classes and triggers must use the `UpperCamelCase` naming convention.